econml/dml.py

```diff
@@ -447,6 +447,34 @@ def __init__(self,
                          n_splits=n_splits,
                          random_state=random_state)
 
+    # override only so that we can update the docstring to indicate support for `StatsModelsInference`
+    def fit(self, Y, T, X=None, W=None, sample_weight=None, sample_var=None, inference='auto'):
```

Comment:
This override is missing the `groups` argument.

Reply:
weird!

Reply:
oooh yeah I got it! It's because now I added this definition of fit to change to 'auto'!